### ISSUE TITLE
Language tabs at top of the site are not accessible via keyboard

### ISSUE THREAD

**Issue by akegan** [bloom-housing/bloom#1455]
We had our a11y expert do an audit of our react homepage and she found that the languages at the top of the page are not accessible via the keyboard. The focus jumps from "skip to main content" to the homepage logo, then to the menu bar. 

I checked housing.acgov.org and it has the same issue. 
![Languages_keyboard_nav](https://user-images.githubusercontent.com/11825994/124309193-34b9e800-db1f-11eb-836b-1cc52f0c49a2.gif)

For folks with no or low vision, they aren't currently able to access the site in another language. 

cc @emilyshore

**Comment by akegan** [comment]
Exygy folks agree that this is a priority.

**Comment by akegan** [comment]
Update from @seanmalbert they might pull it into this next sprint (starting tomorrrow)

**Comment by seanmalbert** [comment]
@kathyccheng ,
Do you think we have capacity to get this into this upcoming sprint? If not, can we prioritize it for the next one?

**Comment by slowbot** [comment]
@seanmalbert I looked at this and it looks like the language options are not currently links which means they will not be detected by the screen reader

![image](https://user-images.githubusercontent.com/13915/125816273-fe40327d-6867-4128-96de-7dff4d827500.png)

**Comment by dominikx96** [comment]
@kathyccheng it has been closed by #1528
